web/pandas/community/license.md

```diff
@@ -0,0 +1,31 @@
+# License
+
```

Comment:
For me, this doesn't add much value as separate page on the website, I would say